cloud/azure/README.md

```diff
@@ -1,7 +1,34 @@
-# Azure Terraform 
+# Azure Terraform Setup
 
-## Overview
-Currently the `main.tf` file includes the main terraform configuration file that sets up the different services that civiform uses in Azure. 
+## create a backend_vars file 
 
-## Variables
-`main.tf` uses variables via `var.var_name`. Those variable definitions and defaults live in `variables.tf`. For values that need to be private create a `.auto.tfvars` or `terraform.tfvars` file which should not be checked into a public repo (as it will often contain private information). We have added an example file to copy and fill out with specific to deployment infromation
+Backend vars should look like the following
+```
+resource_group_name  = < rg group name >
+storage_account_name  = < account name >
+container_name  = < container name >
+key  = "terraform.tfstate"
+```
+You can get the resource group name, the storage account name, and the container name from the azure portal (or check this into a private repo). 
+
+## export your account key
+We use a shared state on an azure blob so that we don't have conflicts with our deploys. In order to get the account_key use the azure CLI. You'll need to get the resource_group_name, storage_account_name from the prior step or from someone else's backend_vars
```

Comment:
suggestion(blocking): can you add a line explaining what this account key value is used for?
```suggestion
In order to get the `account_key` value, use the azure CLI. You'll need to get the resource_group_name, storage_account_name from the prior step.
```